examples/pytorch_profiler/build.envd

```diff
@@ -0,0 +1,16 @@
+envdlib = include("https://github.com/tensorchord/envdlib")
+
+
+def build():
+    base(os="ubuntu20.04", language="python3")
+    shell("zsh")
+    install.cuda(version="11.2.0", cudnn="8")
+    install.python_packages(
+        [
+            "torch",
+            "torchvision",
+            "torch_tb_profiler",
+            "--extra-index-url https://download.pytorch.org/whl/cu113",
+        ]
+    )
+    envdlib.tensorboard(8888, envd_dir="/home/envd/log/efficientnet")
```

Comment:
I think it should be 
```suggestion
    envdlib.tensorboard(8888, envd_dir="/home/envd/log")
```

Not sure why mounting `/home/envd/log/efficientnet` cause `/home/envd/log` owned by the root.

cc @gaocegege

Reply:
Would you mind opening an issue? I can help take a look.

Reply:
https://github.com/tensorchord/envdlib/pull/8 This should fix it. No need to change the dir.

Let me double check the mount permission.

Reply:
Yeah. I can reproduce this. Let me open an issue.